### DISCUSSION TITLE
Activity Log Saves DateTime with Different Timezone

### DISCUSSION THREAD

**Discussion by lyndonjohn** [spatie/laravel-activitylog#822] [Q&A] [answered]
Hello!

In my `products` table, `date_delivered` is saved with the correct format and timezone.
But checking the properties in `activity_log` table, `date_delivered` was saved with a different timezone.

For example, I will select `2020-12-21 13:30:00`, it will be saved in `activity_log`'s properties as `2020-12-21T05:30:00.000000Z`. The time was changed from 1:30 to 5:30.

Am I missing some configuration?

This is my input field:
`<input type="datetime-local" name="date_delivered">`

Thanks.

**Comment by Gummibeer** [comment]
Hey,
in which timezone is your original value? Is it `+08:00`? If so the value is still the same, only in a different timezone.

**Reply by lyndonjohn** [reply]
Thanks for your reply.

Yes, `+8:00`.  I'm just wondering why it's not saving according to my set timezone or to my selected date and time.

**Comment by lyndonjohn** [comment, marked as answer]
My last resort.

I just converted the date according to my timezone to display the correct date and time.
`Carbon\Carbon::parse($property)->setTimezone('Asia/Manila')->format('Y-m-d H:i:s')`
